src/scss/global.scss

```diff
@@ -180,6 +180,24 @@ button.btn {
     margin-left: auto;
     margin-right: auto;
   }
+  &.btn-primary {
+    background-image: none;
+    background-color: $primary;
+
+    &:hover {
+      background-image: none;
+      background-color: $dimmer-primary;
+    }
+  }
+  &.btn-secondary {
```

Comment:
[nit]
```suggestion
  }

  &.btn-secondary {
```

Reply:
Fixed in:https://github.com/sparkletown/sparkle/pull/1377/commits/de659957079dd74136f26d29e9b952be12add510